src/ip.cpp

```diff
@@ -67,4 +48,22 @@ namespace oxen::quic
         return "{}"_format(buf);
     }
 
+    ipv6 ipv6_net::max_ip()
+    {
+        auto b = base.to_base(mask);
+
+        if (mask > 64)
+        {
+            b.hi = base.hi;
+            b.lo |= (128 - mask) - 1;
+        }
+        else
+        {
+            b.hi |= (1 << (64 - mask)) - 1;
+            b.lo = (uint64_t)(int64_t)-1;
```

Comment:
This double cast looks rather weird; I'd suggest instead one of:
```suggestion
            b.lo = std::numeric_limits<uint64_t>::max();
```

```suggestion
            b.lo = static_cast<uint64_t>(-1);
```

Reply:
My mistake, I should have clarified more in my last discord message. This was my last "stab" at the CI fail -- was trying something from SO. It originally did not have the `int64_t`